src/app/Routes.tsx

```diff
@@ -29,177 +31,215 @@ import Preferences from "app/preferences/views/Preferences";
+      element={
+        <ErrorBoundary>
+          <Domains />
+        </ErrorBoundary>
+      }
+      path={`${domainsURLs.details(null, true)}/*`}
+    />
+    <Route
+      element={
         <ErrorBoundary>
           <Images />
         </ErrorBoundary>
-      )}
+      }
+      path={`${imagesURLs.index}/*`}
     />
     <Route
-      path={kvmURLs.kvm}
-      render={() => (
+      element={
         <ErrorBoundary>
           <KVM />
         </ErrorBoundary>
-      )}
+      }
+      path={`${kvmURLs.kvm}/*`}
     />
     <Route
-      path={machineURLs.machines.index}
-      render={() => (
+      element={
         <ErrorBoundary>
           <Machines />
         </ErrorBoundary>
-      )}
+      }
+      path={`${machineURLs.machines.index}/*`}
     />
     <Route
-      path={machineURLs.machine.index(null, true)}
-      render={() => (
+      element={
         <ErrorBoundary>
           <MachineDetails />
         </ErrorBoundary>
-      )}
+      }
+      path={`${machineURLs.machine.index(null, true)}/*`}
     />
     <Route
-      path={poolsURLs.pools}
-      render={() => (
+      element={
         <ErrorBoundary>
           <Pools />
         </ErrorBoundary>
-      )}
+      }
+      path={`${poolsURLs.pools}/*`}
     />
     <Route
-      path={tagURLs.tags.index}
-      render={() => (
+      element={
         <ErrorBoundary>
           <Tags />
         </ErrorBoundary>
-      )}
+      }
+      path={`${tagURLs.tags.index}/*`}
     />
     <Route
-      path={tagURLs.tag.index(null, true)}
-      render={() => (
+      element={
         <ErrorBoundary>
           <Tags />
         </ErrorBoundary>
-      )}
+      }
+      path={`${tagURLs.tag.index(null, true)}/*`}
     />
     <Route
-      path={settingsURLs.index}
-      render={() => (
+      element={
         <ErrorBoundary>
           <Settings />
         </ErrorBoundary>
-      )}
-    />
-    {[
-      subnetsURLs.index,
-      subnetsURLs.fabric.index(null, true),
-      subnetsURLs.space.index(null, true),
-      subnetsURLs.subnet.index(null, true),
-      subnetsURLs.vlan.index(null, true),
-    ].map((path) => (
-      <Route
-        exact
-        key={path}
-        path={path}
-        render={() => (
-          <ErrorBoundary>
-            <Subnets />
-          </ErrorBoundary>
-        )}
-      />
-    ))}
-    {[zonesURLs.index, zonesURLs.details(null, true)].map((path) => (
-      <Route
-        exact
-        key={path}
-        path={path}
-        render={() => (
-          <ErrorBoundary>
-            <Zones />
-          </ErrorBoundary>
-        )}
-      />
-    ))}
-    <Route
-      path={dashboardURLs.index}
-      render={() => (
+      }
+      path={`${settingsURLs.index}/*`}
+    />
+    <Route
+      element={
+        <ErrorBoundary>
+          <SubnetsList />
+        </ErrorBoundary>
+      }
+      path={`${subnetsURLs.index}/*`}
+    />
+    <Route
+      element={
+        <ErrorBoundary>
+          <FabricDetails />
```

Comment:
These components use to be in `Subnets.tsx` but now need to be in this file as they each have their own top level route e.g. /fabric/